DXF: add a limitation that adding indexes to columns of `TIMESTAMP` data type via the DXF is not supported (#15787)
pingcap/docs: tidb-distributed-execution-framework.md

## Limitation

- The DXF can only schedule the distributed execution of one `ADD INDEX` task at a time. If a new `ADD INDEX` task is submitted before the current `ADD INDEX` distributed task has finished, the new task is executed through a transaction.

> - Adding indexes to columns of `TIMESTAMP` data type via the DXF is not supported, because it might lead to inconsistency between the index and the data.

Reviewer: ```suggestion
- Creating indexes on columns with the TIMESTAMP data type through the DXF is not supported, because it might lead to inconsistency between the index and the data.
```

> - Known issue: Adding indexes to columns of data type timestamp via DXF is not supported, as it may lead to inconsistencies between the index and the data.

Reviewer: ```suggestion
- Adding indexes to columns of `TIMESTAMP` data type via the DXF is not supported, because it might lead to inconsistency between the index and the data.
```

> - Creating indexes on columns with the TIMESTAMP data type through the DXF is not supported, because it might lead to inconsistency between the index and the data.

Reviewer 2: ```suggestion
- Adding indexes on columns with the `TIMESTAMP` data type through the DXF is not supported, because it might lead to inconsistency between the index and the data.
```
